egs/wsj/s5/utils/split_scp.pl

```diff
@@ -44,6 +44,7 @@
 $num_jobs = 0;
 $job_id = 0;
 $utt2spk_file = "";
+$one_based = 0;
```

Comment:
would you mind increasing the loop over x to $x <= 3?  (max num-args).  After that I think it's good to merge.  Thanks!

Reply:
Of course. Done!